support/operations.md

```diff
@@ -0,0 +1,54 @@
+# Operations overview
+
+## How we think of Ops
+
+!> We don't have a tiered approach to operations and support, there is no first or second line, we are it.<br/><br/>Instead, we operate what's known as 'Swarm' support model - work with your friends to fix the problem, our users are our service so if they have a problem, we have a problem.<br/><br/>The goal is not to get rid of the ticket, it's to get rid of the problem and that means resolving the issue.
```

Comment:
I understood that we do have a support team which would be the first line?

Reply:
Indeed, I'll amend to reflect service desk - ta